Tools/Scripts/filter-test-logs

```diff
@@ -94,18 +92,24 @@ def filter_webdriver_tests(log_file_name):
             if 'tests ran as expected' in line:
                 print_line = True
                 print(line, end='')
+    return print_line
 
 def main():
     args = parser()
+    log_file = os.path.abspath(f'{args.output}')
     if args.filter_type == 'jsc':
-        filter_jsc_tests(args.output)
+        printed_results = filter_jsc_tests(log_file)
     elif args.filter_type == 'layout':
-        filter_layout_tests(args.output)
+        printed_results = filter_layout_tests(log_file)
     elif args.filter_type == 'scan-build':
-        filter_scan_build(args.output)
+        printed_results = filter_scan_build(log_file)
     elif args.filter_type == 'webdriver':
-        filter_webdriver_tests(args.output)
-
+        printed_results = filter_webdriver_tests(args.output)
+    if not printed_results:
+        with open(log_file, 'r') as raw_output:
+            for l in iter(deque(raw_output, maxlen=500)):
```

Comment:
This will print out the first 500 lines of the log? Would it be more useful to see the last 500 lines instead (looking at https://s3-us-west-2.amazonaws.com/archives.webkit.org/jsc-only-armv7-release-jscore-test/293099@main-49922.txt)?

Reply:
It prints out the last 500 lines, notice the `deque`.